CriticalMass/IBConstructable.swift

```diff
@@ -32,6 +32,18 @@ extension IBConstructable where Self: UIViewController {
     static func fromNib() -> Self {
         return self.init(nibName: nibName, bundle: bundle)
     }
+
+    static var nibName: String {
+        return typeName
+    }
+
+    static var bundle: Bundle {
+        return Bundle(for: Self.self)
+    }
+
+    static var nib: UINib {
+        return UINib(nibName: nibName, bundle: bundle)
+    }
```

Comment:
yeah. wanted that on viewcontrollers a couple of times already

Reply:
Do we want to go Swift 5.1 and omit the return statements? I also had to confirm UIviewController to the TypeNameProtocol